[rospy.client][INFO] 2021-09-09 16:11:05,253: init_node, name[/server_endpoint], pid[3797]
[xmlrpc][INFO] 2021-09-09 16:11:05,253: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-09 16:11:05,253: Started XML-RPC server [http://wng:45497/]
[rospy.init][INFO] 2021-09-09 16:11:05,253: ROS Slave URI: [http://wng:45497/]
[rospy.impl.masterslave][INFO] 2021-09-09 16:11:05,253: _ready: http://wng:45497/
[xmlrpc][INFO] 2021-09-09 16:11:05,254: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-09-09 16:11:05,254: Registering with master node http://localhost:11311
[rospy.init][INFO] 2021-09-09 16:11:05,354: registered with master
[rospy.rosout][INFO] 2021-09-09 16:11:05,354: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-09 16:11:05,357: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-09 16:11:05,359: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-09 16:11:05,369: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-09 16:11:05,578: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-09 16:11:38,846: Connection from 192.168.1.146
[rosout][INFO] 2021-09-09 16:11:38,851: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-09 16:11:38,895: topic[/execution_check] adding connection to [http://wng:45497/], count 0
[rospy.internal][INFO] 2021-09-09 16:11:38,896: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-09 16:42:50,617: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-09 16:42:50,620: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-09 16:42:50,620: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-09 16:42:50,620: topic[/execution_check] removing connection to http://wng:45497/
[rospy.topics][ERROR] 2021-09-09 16:42:50,621: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 846, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2021-09-09 16:42:50,621: signal-2
[rospy.internal][WARNING] 2021-09-09 16:42:50,621: Unknown error initiating TCP/IP socket to wng:38283 (http://wng:45497/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2021-09-09 16:42:50,787: signal_shutdown [atexit]
